README.md

```diff
@@ -86,10 +86,20 @@ paramount**. `simpleCache` assumes that your name for an object is a perfect
 identifier for that object; in other words, don't cache things that you plan to
 change.
 
+### Highlights of exported functions
+
+- `simpleCache()`: Caches and reloads objects created based on instruction
```

Comment:
I would say, maybe, something like "Creates and caches or reloads cached results of provided R instruction code"